nccl/common/prep_ami.sh

```diff
@@ -220,8 +220,12 @@ case $PLATFORM_ID in
         install_software
         ;;
     ubuntu)
-        # Wait for initialization to complete
-        sleep 100
+        # Wait until lock /var/lib/dpkg/lock-frontend released
+        sleep 300
+        # Building aws-ofi-nccl plugin with openmpi throws the following error:
+        # /usr/bin/ld: cannot find -ludev
+        # Installing libudev-dev mitigates the issue
+        sudo apt-get install -y libudev-dev
```

Comment:
We should not need this anymore. The latest EFA installer fixed this issue.

cc: @zhngaj

Reply:
For our testing we are not using the lates installer but instead using which come with DL amis.